cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -67,6 +67,8 @@ type ScaleUpRequest struct {
 	ExpectedAddTime time.Time
 	// How much the node group is increased.
 	Increase int
+	// NodeGroupName is the node group to be scaled up.
+	NodeGroupName string
```

Comment:
Is there any reason for storing this if we already have the NodeGroup? The assumption is NodeGroup.Id() should not make any costly operations like API calls (or cache them internally) - we call it all over CA code.